Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -92,18 +95,23 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
         switch command {
         // View commands
         case let command as RUMStartViewCommand where command.identity === identity:
+            if didReceiveStartCommand {
+                // This is the case of duplicated "start" command. We know that the Session scope has created another instance of
+                // the `RUMViewScope` for tracking this View, so we mark this one as inactive.
+                isActiveView = false
```

Comment:
Not sure I understand. "Duplicated views case" is actually something we want. If one View is waiting for its Resources to complete, but another one is started, we have two running `RUMViewScope`, which is totally fine. We just mark the prior as `isActiveView == false` so it doesn't capture new Resources anymore.

Reply:
the case in my mind was calling `startView` twice for some reason (eg: once in `viewDidLoad` once in `viewWillAppear`) then we will have duplicate views, is this desired?

Reply:
Yes, this would mean bad instrumentation. 

Please note, that it's also fine to receive:
```swift
rum.startView(vc1)
rum.stopView(vc1)
rum.startView(vc1)
rum.stopView(vc1)
rum.startView(vc2)
rum.stopView(vc2)
```
and result with 3 running `RUMViewScopes` at a time:
![Simulator Screen Shot - iPhone 11 - 2020-08-28 at 19 38 01](https://user-images.githubusercontent.com/2358722/91824588-4ac6f580-ec3b-11ea-9c9f-4365986f9801.png)
if both instances of `vc1` are still waiting for Resources.

If we'd like the `RUMSessionScope` to mark and manage the "active" View, it would have to know how many Resources are pending on each instance of `RUMViewScope` and close that scope if it goes down to `0`. This would mean that `viewScope.pendingResourcesCount` would have to be accessed by `RUMSessionScope` and that the result of `viewScope.process(command:)` should be ignored by the Session scope. Both things violate the unidirectional architecture of scopes.